src/broker/cas.c

```diff
@@ -673,6 +674,7 @@ shard_cas_main (void)
     req_info.need_rollback = TRUE;
 
     gettimeofday (&tran_start_time, NULL);
+    cub_ddl_log_start_time (&tran_start_time);
```

Comment:
Shouldn't you have to call the **cub_ddl_log_init()** function before calling this function? It is inside shard_cas_main ().

```
cub_ddl_log_init();
...
cub_ddl_log_start_time (&tran_start_time);
```

Reply:
Thank you, can't verify ddl audit in shard_cas_main(). So I will delete it.